src/DataStructures/ComplexMatrix.hpp

```diff
@@ -0,0 +1,25 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <complex>
+
+#include "DataStructures/DynamicMatrix.hpp"
+
+/*!
+ * \ingroup DataStructuresGroup
+ * \brief A dynamically sized matrix of `std::complex<double>`s with
+ * column-major storage.
+ *
+ * \note This is a thin wrapper around `blaze::DynamicMatrix`. Please refer to
+ * [Blaze documentation](https://bitbucket.org/blaze-lib/blaze/wiki/Matrices)
+ * for information on how to use it.
+ */
+class ComplexMatrix
+    : public blaze::DynamicMatrix<std::complex<double>, blaze::columnMajor> {
```

Comment:
I think we can just use `blaze::DynamicMatrix` directly, no need for this wrapper. I got rid of the other wrappers in #3832 and only left `Matrix` to avoid refactoring a lot of code (I would have replaced `Matrix` with a type alias to `blaze::DynamicMatrix<double, colMajor>` but you can't forward-declare a type alias).